src/com/nutrons/framework/controllers/Events.java

```diff
@@ -34,6 +36,16 @@ public void actOn(Talon talon) {
   public static ControllerEvent pid(double pval, double ival, double dval, double fval) {
     return new LoopPropertiesEvent(pval, ival, dval, fval);
   }
+  public static ControllerEvent combine(ControllerEvent... Events) {
+    return new ControllerEvent() {
+      @Override
+      public void actOn(Talon talon) {
+        for (ControllerEvent E : Events) {
```

Comment:
Use camel case variable names.  The first letter should be lowercase for everything but class names.  Then everything successive word should be capitalized.  Also, the google style guide does not allow for single letter variable names.